test/functional/legacy/089_number_relnumber_findfile_spec.lua

```diff
@@ -1,116 +0,0 @@
--- - Some tests for setting 'number' and 'relativenumber'
```

Comment:
Normally I recommend keeping the Lua-converted version of the legacy test, but this one is simple enough that it probably isn't useful.

Reply:
@justinmk Ok, I will keep lua-converted version next time.